test/extensions/resource_monitors/fixed_heap/fixed_heap_monitor_test.cc

```diff
@@ -39,6 +41,30 @@ class ResourcePressure : public Server::ResourceUpdateCallbacks {
 };
 
 TEST(FixedHeapMonitorTest, ComputesCorrectUsage) {
+
+  TestScopedRuntime scoped_runtime;
+  scoped_runtime.mergeValues({{"envoy.reloadable_features.do_not_count_mapped_pages_as_free", "false"}});
+
+  envoy::extensions::resource_monitors::fixed_heap::v3::FixedHeapConfig config;
+  config.set_max_heap_size_bytes(1000);
+  auto stats_reader = std::make_unique<MockMemoryStatsReader>();
+  EXPECT_CALL(*stats_reader, reservedHeapBytes()).WillOnce(testing::Return(800));
```

Comment:
using `testing::Return` outside the test since we reference this a lot in this file

Reply:
Resolved